net/databag/files/databag.init

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh /etc/rc.common
+ 
+START=10
```

Comment:
`10` might be too early for a user program.

and please convert it to procd daemon if possible